integration/client_server_test.go

```diff
@@ -220,6 +223,15 @@ func TestClientServer(t *testing.T) {
 			},
 			golden: "testdata/busybox-with-lockfile.json.golden",
 		},
+		{
+			name: "scan pox.xml with fs command in client/server mode",
+			args: csArgs{
+				Command:          "fs",
+				RemoteAddrOption: "--server",
+				Target:           "testdata/fixtures/fs/pom/",
+			},
+			golden: "testdata/fs-pom.json.golder",
```

Comment:
We have to use the same golden file with standalone mode. Maybe we have some differences now?

Reply:
Great idea! done.